src/components/SecondaryPanes/Frames/index.js

```diff
@@ -154,7 +154,7 @@ class Frames extends Component {
       ? L10N.getStr("callStack.collapse")
       : L10N.getStr("callStack.expand");
 
-    frames = collapseFrames(frames);
+    frames = this.collapseFrames(frames);
```

Comment:
Was the issue that if frameworkGroupingOn was not checked, collapse frame would be run with unexpected results?